department-of-veterans-affairs/caseflow
# AMA attorney checkout flow works

Connects #https://github.com/department-of-veterans-affairs/caseflow/issues/6778

### Testing Plan
1. Log in as `BVASCASPER1`
2. Navigate to `/queue` and select an AMA appeal
3. Notice that OMO request flow is hidden
4. Navigate through Attorney Draft Decision flow
5. Select "Grant" or "Deny" for all issues
6. Complete checkout flow
7. Find judge you selected in the flow, run `JudgeQueue.new(user: user).tasks` and verify that the appeal is now assigned to that judge
